packages/synapse-interface/components/Maintenance/Events/template/MaintenanceEvent.tsx

```diff
@@ -112,3 +136,99 @@ export const useMaintenanceCountdownProgress = () => {
 const isChainIncluded = (chainList: number[], hasChains: number[]) => {
   return hasChains.some((chainId) => chainList.includes(chainId))
 }
+
+/** Aggregators */
+
+interface ChainPause {
+  id: string
+  pausedChains: number[]
+  startTime: Date
+  endTime: Date
+  bannerStartTime: Date
+  bannerEndTime: Date
+  warningMessage: any
+  bannerMessage: any
+  progressBarMessage: any
+}
+
+const PAUSED_CHAINS: ChainPause[] = [
```

Comment:
We need to handle each of these scenarios

- chain is paused on origin side
- chain is paused on destination side (like Dogechain in prod right now, where a user can bridge from Dogechain but not bridge to Dogechain)
- chain is paused on both sides

Reply:
Will need to remove Doge from `PAUSED_TO_CHAIN_IDS` in constants to test this